test/Others.ts

```diff
@@ -2,30 +2,22 @@ import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
 import { expect } from "chai";
```

Comment:
Do these tests work now? Last time I run them, they weren't working.

Reply:
I tested before pushing this time and they worked.  I remember they worked (for me) last time too.

Reply:
Can you please enable them in CI then? Update the Github workflow to uncomment the part I have commented out.